scripts/steamed_hams.py

```diff
@@ -112,25 +113,45 @@ def steamed_hams_parser():
     """
 
     parser = argparse.ArgumentParser(usage=usage_msg())
-    parser.add_argument("experiment_module", help="The name of the module in the experiments directory "
-                                                  "that contains your Experiment class, "
-                                                  "e.g. 'normalscan'")
-    parser.add_argument("run_mode", help="The mode to run, switches scons builds and some arguments to "
-                                         "modules based on this mode. Commonly 'release'.")
-    parser.add_argument("scheduling_mode_type", help="The type of scheduling time for this experiment "
-                                                     "run, e.g. 'common', 'special', or 'discretionary'.")
-    parser.add_argument("--embargo", action="store_true", help="Embargo the file (makes the CPID negative)")
-    parser.add_argument("--kwargs", nargs='+', default='',
-                        help="Keyword arguments for the experiment. Each must be formatted as kw=val")
-    parser.add_argument("--realtime-off", action="store_true",
-                        help="Disable the realtime FITACF3 and data server module")
+    parser.add_argument(
+        "experiment_module",
+        help="The name of the module in the experiments directory "
+        "that contains your Experiment class, "
+        "e.g. 'normalscan'",
```

Comment:
Some of the formatting is a little wonky, like this - is there a linter to combine strings that don't need to be multiline?

Reply:
This is wonky because the string was originally split in the same places. Black doesn't combine strings together. From a bit of searching, what you are asking for doesn't exist. Black also doesn't split strings or comments if they are over the line length.